ckan/logic/action/get.py

```diff
@@ -3532,22 +3532,36 @@ def job_list(context, data_dict):
     :param list queues: Queues to list jobs from. If not given then the
         jobs from all queues are listed.
 
+    :param int limit: Number to limit the list of jobs by.
+
+    :param bool ids_only: Whether to return only a list if job IDs or not.
+
     :returns: The currently enqueued background jobs.
     :rtype: list
 
+    Will return the list in the way that RQ workers will execute the jobs.
+    Thus the left most non-empty queue will be emptied firts
+    before the next right non-empty one.
+
     .. versionadded:: 2.7
     '''
     _check_access(u'job_list', context, data_dict)
-    dictized_jobs = []
+    jobs_list = []
     queues = data_dict.get(u'queues')
+    limit = data_dict.get(u'limit',
```

Comment:
Probably better to validate the parameters and return proper validation errors, but here you at least need to make sure this is an int